e2e/Cargo.toml

```diff
@@ -0,0 +1,43 @@
+# You must change these to your own details.
+[package]
+name = "e2e"
+description = "E2E testing the source compiler"
+version = "0.1.0"
+authors = ["Mohammad Yousuf Minhaj Zia <yzia2000@gmail.com>"]
+categories = ["wasm"]
+readme = "README.md"
+edition = "2018"
+
+[lib]
+crate-type = ["cdylib"]
+
+[features]
+# If you uncomment this line, it will enable `wee_alloc`:
+#default = ["wee_alloc"]
+
+[dependencies]
+source-compiler = { path = "../source-compiler" }
+js-sys = "0.3.22"
+wasm-bindgen = "0.2.45"
+wasm-bindgen-futures = "0.4"
+
+wee_alloc = { version = "0.4.2", optional = true }
+
+# The `web-sys` crate allows you to interact with the various browser APIs,
+# like the DOM.
+[dependencies.web-sys]
+version = "0.3.22"
+features = ["console"]
+
+# The `console_error_panic_hook` crate provides better debugging of panics by
+# logging them with `console.error`. This is great for development, but requires
+# all the `std::fmt` and `std::panicking` infrastructure, so it's only enabled
+# in debug mode.
+[target."cfg(debug_assertions)".dependencies]
+console_error_panic_hook = "0.1.5"
+
+# These crates are used for running unit tests.
+[dev-dependencies]
+wasm-bindgen-test = "0.2.45"
+futures = "0.1.27"
+js-sys = "0.3.22"
```

Comment:
I don't think you need to repeat a dev-dependency that is already a normal dependency.